### PR TITLE
Script for looking at model accuracy on pro positions

### PR DESCRIPTION
**Pull request by jmgilmer**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -8,3 +8,5 @@ protobuf
 six
 petname
 google.cloud.logging
+matplotlib
```
Comment (amj) [inline requirements.txt]: these are some heavy dependencies; matplotlib in particular will probably give folks headaches.  Could we maybe make this 'requirements-analysis.txt' so folks don't necessarily need to add them all to run the oneoffs

File: sgf/fetch_dataset.sh [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```
Comment (amj) [inline sgf/fetch_dataset.sh]: Can you put a comment in here that these games are public domain?

File: training_curve.py [hunk 3]
```diff
@@ -0,0 +1,250 @@
+"""
```
Comment (amj) [inline training_curve.py]: can we move this file to the oneoffs/ directory?

File: training_curve.py [hunk 4]
```diff
@@ -0,0 +1,250 @@
...
+  if props.get('KM') is None:
+    return False
+  return props.get('KM')[0] == komi_str
+
+def find_and_filter_sgf_files(base_dir, min_year = None, komi = None):
+  base_dir = "sgf/baduk_db"
+  sgf_files = []
+  count = 0
+  print("Finding all sgf files in {} with year >= {} and komi = {}".format(base_dir, min_year, komi))
+  for i, (dirpath, dirnames, filenames) in enumerate(os.walk(base_dir)):
```
Comment (amj) [inline training_curve.py]: recommend using tqdm for ez progress bars (here and for all for loops that take longer than 10s :)

File: sgf/fetch_dataset.sh (hunk 2 above)
Comment (jmgilmer) [reply sgf/fetch_dataset.sh]: Done

File: requirements.txt (hunk 1 above)
Comment (jmgilmer) [reply requirements.txt]: Done

File: training_curve.py (hunk 3 above)
Comment (jmgilmer) [reply training_curve.py]: Done

File: training_curve.py (hunk 4 above)
Comment (jmgilmer) [reply training_curve.py]: Done

File: oneoffs/training_curve.py [hunk 5]
```diff
@@ -0,0 +1,251 @@
...
+from tqdm import tqdm
+from sgf_wrapper import sgf_prop
+from gtp_wrapper import make_gtp_instance, MCTSPlayer
+from utils import logged_timer as timer
+from tensorflow import gfile
+
+tf.app.flags.DEFINE_string(
+    "sgf_dir", "sgf/baduk_db/", "sgf database")
+
+tf.app.flags.DEFINE_string("model_dir", "gs://mugozero-v1/models",
```
Comment (amj) [inline oneoffs/training_curve.py]: nit: remove bucket name, practice is to make a dir called 'saved_models'

Comment (amj) [reply oneoffs/training_curve.py]: i'll do it, nm

**Pull request merged**